app/pages/sections/order-items/catalogue-solutions/select/routes.js

```diff
@@ -141,12 +150,11 @@ export const catalogueSolutionsSelectRoutes = (authProvider, addContext, session
       return res.redirect(`${config.baseUrl}/organisation/${orderId}/catalogue-solutions/neworderitem`);
     }
 
-    const solutionId = sessionManager.getFromSession({ req, key: 'selectedSolutionId' });
-    const solutionData = await getSolution({ solutionId });
+    const solutionName = sessionManager.getFromSession({ req, key: 'selectedItemName' });
```

Comment:
Was thinking the variable name could match the key but due to the context of the test this is fine 👍

Reply:
I'll do this one later. Will leave for now but it's a valid point